packages/components/src/item-group/stories/index.story.tsx

```diff
@@ -83,6 +87,8 @@ NonClickableItems.args = {
 
 export const CustomItemSize: StoryFn< typeof ItemGroup > = Template.bind( {} );
 CustomItemSize.args = {
+	isBordered: true,
+	isSeparated: true,
```

Comment:
Same as above

Reply:
Addressed as part of 5e26b6fe8939cc87cabc95c368eaca1757e24074